Description: Adding Arale device id (mtp and mtp+adb)
Forwarded: not-needed
Author: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>

This is the temporary id used by this device, once the final
one is in place, it'll be forwarded to upstream.

---
 src/music-players.h |    4 ++++
 1 file changed, 4 insertions(+)

--- libmtp.orig/src/music-players.h
+++ libmtp/src/music-players.h
@@ -2117,6 +2117,10 @@
       DEVICE_FLAGS_ANDROID_BUGS },
   { "bq", 0x2a47, "Krillin (MTP+ADB)", 0x0c02,
       DEVICE_FLAGS_ANDROID_BUGS },
+  { "Arale", 0x2a45, "Arale (MTP)", 0x2008,
+      DEVICE_FLAGS_ANDROID_BUGS },
+  { "Arale", 0x2a45, "Arale (MTP+ADB)", 0x0c02,
+      DEVICE_FLAGS_ANDROID_BUGS },
 
 
   /*
